catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerTest.java

```diff
@@ -183,11 +183,11 @@ protected AdminSourceHelper getHelper() {
                 // Mock out the metatypes
```

Comment:
It's possible, but if the configuration was adjusted, how would the test behave. Would we be testing the default - that Schematron should be filtered out - or that whatever was in excluded was in fact filtered out?